hackathon/views.py

```diff
@@ -188,3 +190,46 @@ def delete_hackathon(request, hackathon_id):
 
     messages.success(request, f'{hackathon.display_name} has been successfully deleted!')
     return redirect("hackathon:hackathon-list")
+
+class HackathonDetailView(DetailView):
+    """Renders a page with Hackathon details."""
+    model = Hackathon
+    context_object_name = "hackathon"
+
+
+def ajax_enroll_toggle(request):
+    """Swaps between being enrolled as a judge and unenrolling."""
+    # Checks to make sure the user has the permissions to enroll
```

Comment:
This comment is not needed. Also, in general you don't need to comment every condition. The code itself should be readable and only on special occasions you should add comments if extra context is necessary. Please remove also the comments below, I think it should be fine to understand what you are trying to achieve.

Reply:
Did you not leave another comment on removing 'ajax' in the view name and url?